### PR TITLE
Add dojo article

### PR DESCRIPTION
**Pull request by Gautier**
photos from Salim are not committed yet as I need to first ask permission to the attendees before putting them out on the internet.

### CODE REVIEW

File: content/articles/ldnpydojos07e02.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+I'll report on the event in this post.
+<!-- PELICAN_END_SUMMARY -->
+
+<img width="100%" src="../images/dojocrowd.jpg">
+
+## What is a Code Dojo?
+
+You can find some information on the [London Python Code Dojo website](http://ldnpydojo.org.uk/).
+
+A Code Dojo is a "safe place to to deliberately practise and develop your
```
Comment (Keats) [inline content/articles/ldnpydojos07e02.md]: 2 to

File: content/articles/ldnpydojos07e02.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+coding skills, and perhaps learn something new."
+
+It originally comes from Paris (like most good things) and is based on the
```
Comment (Keats) [inline content/articles/ldnpydojos07e02.md]: lies

File: content/articles/ldnpydojos07e02.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+repository](https://github.com/jamesbroadhead/Axelrod/tree/master/axelrod/strategies).
+
+### The raffle
+
+O'Reilly usually provides us with a book to give away to an attendee in a
+raffle but this month they provided us with a free two days pass at
+[OSCON](http://www.oscon.com/open-source-eu-2015)!
+
+If you came to the dojo, you can use the discount code "ldnpydojo20" and get
+20% off for OSCON passes.
```
Comment (Keats) [inline content/articles/ldnpydojos07e02.md]: not sure you should put that in the article

File: content/articles/ldnpydojos07e02.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+    def strategy(self, opponent):
+        opponent.history[:] = ['C'] * len(opponent.history)
+        return 'D'
+```
+
+It rewrites the opponent history to make it think it always cooperate but
+actually only defects!
+
+You can find the other strategies we wrote on the [git
+repository](https://github.com/jamesbroadhead/Axelrod/tree/master/axelrod/strategies).
```
Comment (Keats) [inline content/articles/ldnpydojos07e02.md]: no mention of team 2, best team ;(

File: content/articles/ldnpydojos07e02.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+<h4>Prepared kata</h4>
+
+<p>
+A presenter shows how to solve the challenge from scratch, using TDD and
+BabySteps.
+Each step must make sense to everyone present for.
+People should interrupt only if they don't understand what is going on.
+</p>
+
+<h4>Randori kata</h4>
```
Comment (Keats) [inline content/articles/ldnpydojos07e02.md]: never heard of that

File: content/articles/ldnpydojos07e02.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+## What is a Code Dojo?
+
+You can find some information on the [London Python Code Dojo website](http://ldnpydojo.org.uk/).
+
+A Code Dojo is a "safe place to to deliberately practise and develop your
+coding skills, and perhaps learn something new."
+
+It originally comes from Paris (like most good things) and is based on the
+following ideas:
```
Comment (Keats) [inline content/articles/ldnpydojos07e02.md]: that doesn't really look like the kind of dojo we do ?

File: content/articles/ldnpydojos07e02.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+- 5 minutes: splitting into teams
+- 90 minutes: working in teams
+- 20 minutes: teams show and tell
+- 5 minutes: raffle
+- followed by the pub!
+
+The teams are randomly assigned so members have a variety of levels.
+As a junior developer your objective is to be on the keyboard and write code as
+much as possible.
+And as a more senior developer your objective is to help the more junior
```
Comment (Keats) [inline content/articles/ldnpydojos07e02.md]: these 2 sentences don't flow nicely, maybe make them into one?

File: content/articles/ldnpydojos07e02.md [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+interactive format:
+
+- ~1 hour: social time with food and drinks, we put down ideas on a whiteboard
+- 5 minutes: Code Dojo presentation and Python community annoucements
+- 10 minutes: voting on the ideas
+- 5 minutes: splitting into teams
+- 90 minutes: working in teams
+- 20 minutes: teams show and tell
+- 5 minutes: raffle
+- followed by the pub!
```
Comment (Keats) [inline content/articles/ldnpydojos07e02.md]: i'd have only: pub!

File: content/articles/ldnpydojos07e02.md (hunk 1 above)
Comment (Gautier) [inline content/articles/ldnpydojos07e02.md]: fixed

File: content/articles/ldnpydojos07e02.md (hunk 2 above)
Comment (Gautier) [inline content/articles/ldnpydojos07e02.md]: Ah ah! I didn't plan on leaving this for the actual article :)

File: content/articles/ldnpydojos07e02.md (hunk 6 above)
Comment (Gautier) [inline content/articles/ldnpydojos07e02.md]: the format is different yes, but the premise and characteristics are the same.

File: content/articles/ldnpydojos07e02.md (hunk 7 above)
Comment (Gautier) [inline content/articles/ldnpydojos07e02.md]: tried a different way

File: content/articles/ldnpydojos07e02.md (hunk 8 above)
Comment (Gautier) [inline content/articles/ldnpydojos07e02.md]: changed

File: content/articles/ldnpydojos07e02.md (hunk 3 above)
Comment (Gautier) [inline content/articles/ldnpydojos07e02.md]: I'll put that in the email asking for permission for the photos.

File: content/articles/ldnpydojos07e02.md (hunk 4 above)
Comment (Gautier) [inline content/articles/ldnpydojos07e02.md]: It is, indeed, a partial post.

File: content/articles/ldnpydojos07e02.md [hunk 9]
```diff
@@ -0,0 +1,175 @@
+Title: The London Python Dojo S07E02
+Date: 2015-10-03
+Short_summary: The We Are Wizards team was at the last London Python Code Dojo.
+Category: Events
+Authors: Gautier
+
+
+The We Are Wizards team was present at the last London Python Code Dojo held at
+Bank of America in Canary Wharf on the first of October.
```
Comment (teh) [inline content/articles/ldnpydojos07e02.md]: This sounds incredibly formal :)

Maybe We (wizards) attended the London ... at BAML at .. . Find our report inside.

?

File: content/articles/ldnpydojos07e02.md [hunk 10]
```diff
@@ -0,0 +1,175 @@
...
+I'll report on the event in this post.
+<!-- PELICAN_END_SUMMARY -->
+
+<img width="100%" src="../images/dojocrowd.jpg">
+
+## What is a Code Dojo?
+
+You can find some information on the [London Python Code Dojo website](http://ldnpydojo.org.uk/).
+
+A Code Dojo is a "safe place to deliberately practise and develop your
```
Comment (teh) [inline content/articles/ldnpydojos07e02.md]: Make this a block quote with ">" ?

File: content/articles/ldnpydojos07e02.md [hunk 11]
```diff
@@ -0,0 +1,175 @@
...
+coding skills, and perhaps learn something new."
+
+It originally comes from Paris and is based on the following ideas:
```
Comment (teh) [inline content/articles/ldnpydojos07e02.md]: "It originated in Paris"

File: content/articles/ldnpydojos07e02.md [hunk 12]
```diff
@@ -0,0 +1,175 @@
...
+<li>all skill levels are welcome</li>
+<li>safe to try new ideas</li>
+</ul>
+
+<h4>Types of meeting</h4>
+
+<h4>Prepared kata</h4>
+
+<p>
+A presenter shows how to solve the challenge from scratch, using TDD and
```
Comment (teh) [inline content/articles/ldnpydojos07e02.md]: "baby steps" ? Not sure why you chose this capitalization?

Comment (teh) [inline content/articles/ldnpydojos07e02.md]: nevermind

File: content/articles/ldnpydojos07e02.md [hunk 13]
```diff
@@ -0,0 +1,175 @@
...
+At the end of the timebox, the driver goes back to the audience, the copilot
+becomes driver and one of the audience step up to be copilot.
+</p>
+
+</blockquote>
+
+
+### The London style
+
+In London we start from the same premise and characteristics but we have a more
```
Comment (teh) [inline content/articles/ldnpydojos07e02.md]: "with the same premise, sticking to the characteriscs" - premise and characteristics don't have the same "type" so can't be used like this :)

File: content/articles/ldnpydojos07e02.md [hunk 14]
```diff
@@ -0,0 +1,175 @@
...
+
+O'Reilly usually provides us with a book to give away to an attendee in a
+raffle but this month they provided us with a free two days pass at
+[OSCON](http://www.oscon.com/open-source-eu-2015)!
+
+## Conclusion
+
+<img width="100%" src="../images/dojoteam.jpg">
+
+We had a fun night and learned that in the prisoner's dilemma, short of
```
Comment (teh) [inline content/articles/ldnpydojos07e02.md]: is to "always defect" ?

Comment (teh) [inline content/articles/ldnpydojos07e02.md]: also depends on the payoff matrix but this post is probably not the right place to go into that :)

File: content/articles/ldnpydojos07e02.md [hunk 15]
```diff
@@ -0,0 +1,175 @@
...
+```python
+class Fucker(Player):
+    name = 'Fucker'
+
+    def strategy(self, opponent):
+        opponent.history[:] = ['C'] * len(opponent.history)
+        return 'D'
+```
+
+It rewrites the opponent history to make it think it always cooperate but
```
Comment (teh) [inline content/articles/ldnpydojos07e02.md]: it always cooperates (missing s)

**Pull request merged**